### PR TITLE
Moving MD5 hashing into it's own file

### PR DESCRIPTION
**Pull request by bnbowman**

### CODE REVIEW

File: include/pbbam/MD5.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2014-2015, Pacific Biosciences of California, Inc.
```
Comment (pb-cdunn) [inline include/pbbam/MD5.h]: You can put the license into a `LICENSE` file. That's SOP in GitHub. Then you don't have to include it in every file. Just FYI.

Comment (dalexander) [inline include/pbbam/MD5.h]: This is per the PacBio style guide, Chris.

**Comment by pezmaster31** [comment]
Applied in p4 (CL 181567) & published to repo here.

Thanks @bnbowman.

**Comment by pezmaster31** [comment]
@bnbowman: please note I did implement this in a .cpp file, so our edits are *not* 1-for-1. But the API is exactly compatible

**Pull request closed without merging**